contributing-guides/translation-templates/common-arguments.md

```diff
@@ -31,7 +31,7 @@ Only the left-alignment of the header gets lost and has to be readded again (`|-
 | sh    |                      |                        |                                   |           |                   |
 | sr    |                      |                        |                                   |           |                   |
 | sv    |                      |                        |                                   |           |                   |
-| ta    |                      |                        |                                   |           |                   |
+| ta    | கோப்பு               | அடைவு                | கோப்போ/அடைவோ                | நிரல்தொகுப்பு | பயனர்ப்பெயர்   |
```

Comment:
Did you fill this in per hand or use [tableconvert.com](https://tableconvert.com/) for this? The alignment doesn't look right, but that could also be due to the full-width characters.